pkg/maps/ctmap/gc/gc.go

```diff
@@ -30,7 +31,8 @@ type EndpointManager interface {
 }
 
 // Enable enables the connection tracking garbage collection.
-func Enable(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint, mgr EndpointManager) {
+func Enable(ipv4, ipv6 bool, restoredEndpoints []*endpoint.Endpoint, mgr EndpointManager,
+	nodeAddressing types.NodeAddressing) {
```

Comment:
A bit of a nit but it would be nice to comment or document which of these arguments are used for GC filtering.